### PR TITLE
[IMP] spreadsheet: update o_spreadsheet to latest version

### PR DESCRIPTION
**Pull request by LucasLefevre**
### Contains the following commits:

https://github.com/odoo/o-spreadsheet/commit/b04cf78c2 [REL] 16.4.0-alpha.7
https://github.com/odoo/o-spreadsheet/commit/37e6ebed5 [IMP] helpers: add ast iteration helper Task: 3360232
https://github.com/odoo/o-spreadsheet/commit/c44431ffa [IMP] TopBarComposer: allow selection in readonly mode Task: 3378521
https://github.com/odoo/o-spreadsheet/commit/44063ddaf [ADD] functions: add TRUE and FALSE boolean functions Task: 3360232
https://github.com/odoo/o-spreadsheet/commit/62a93f842 [REF] tokenizer: decouple tokenizer from functions Task: 3360232
https://github.com/odoo/o-spreadsheet/commit/153a50d23 [IMP] xlsx: import images

### CODE REVIEW

**Comment by LucasLefevre** [comment]
fw-bot ignore

File: addons/spreadsheet/static/src/helpers/model.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 import { DataSources } from "@spreadsheet/data_sources/data_sources";
-import * as spreadsheet from "@odoo/o-spreadsheet";
+import { Model, parse, helpers, iterateAstNodes } from "@odoo/o-spreadsheet";
 import { migrate } from "@spreadsheet/o_spreadsheet/migration";
 import { _t } from "@web/core/l10n/translation";
 import { loadJS } from "@web/core/assets";
 
-const { Model, tokenize } = spreadsheet;
-const { toCartesian } = spreadsheet.helpers;
+const { toCartesian } = helpers;
```
Comment (hokolomopo) [inline addons/spreadsheet/static/src/helpers/model.js:8]: We should probably do that il all the files at one point

**Review by hokolomopo — COMMENTED** [review]
robodoo r+

**Comment by LucasLefevre** [comment]
robodoo rebase-ff

**Pull request closed without merging**